app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
```

Comment:
So following the note in your facade, where we created an array that holds both the foods from our keyword search and the total_hits, we could access those via one call like such:
```
results = FoodFacade.get_list(params[:q])
@foods = results[1]
@total_hits = results[0]
```